drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -1212,8 +1213,14 @@ def _get_item_stats(self):
         """
         items = self._get_item_raw_stats()
 
-        correct_count = len(items.correctly_placed) + len(items.decoy_in_bank)
-        total_count = len(items.required) + len(items.decoy)
+        correct_count = len(items.correctly_placed)
+        total_count = len(items.required)
+
+        if hasattr(self.runtime, 'course_id') and get_config_waffle_flag().is_enabled(self.runtime.course_id):
```

Comment:
Please use `self.scope_ids.usage_id.context_key` instead of `self.runtime.course_id`. The latter [is deprecated](https://github.com/openedx/edx-platform/pull/30715) in newer releases.